tests/tests_deployment/test_jupyterhub_api.py

```diff
@@ -9,6 +9,7 @@
 
 
 @pytest.mark.filterwarnings("ignore::urllib3.exceptions.InsecureRequestWarning")
+@pytest.mark.filterwarnings("ignore::DeprecationWarning")
```

Comment:
Is this needed?  Seems like the deployment tests are passing on other branches without this

Reply:
As it turns out these filters are used in few more deployment tests: eg [test_jupyterhub_ssh](https://github.com/nebari-dev/nebari/blob/749a1f14e2fff79bfc107f06dc7087d8727ddd5b/tests/tests_deployment/test_jupyterhub_ssh.py#L72), [test_dask_gateway](https://github.com/nebari-dev/nebari/blob/749a1f14e2fff79bfc107f06dc7087d8727ddd5b/tests/tests_deployment/test_dask_gateway.py#L25)
```
@pytest.mark.filterwarnings("ignore::urllib3.exceptions.InsecureRequestWarning")
@pytest.mark.filterwarnings("ignore::DeprecationWarning")
```
so I am simply removing the TODO.